SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -23,14 +23,32 @@ struct ConfigureConnection
 		: Config(InConfig)
 		, Params()
 		, WorkerType(*Config.WorkerType)
-		, ProtocolLogPrefix(*FormatProtocolPrefix())
+		, WorkerLogPrefix(*FormatLogFilePrefix())
 	{
 		Params = Worker_DefaultConnectionParameters();
 
 		Params.worker_type = WorkerType.Get();
 
-		Params.enable_protocol_logging_at_startup = Config.EnableProtocolLoggingAtStartup;
-		Params.protocol_logging.log_prefix = ProtocolLogPrefix.Get();
+		Logsink.logsink_type = WORKER_LOGSINK_TYPE_ROTATING_FILE;
+		Logsink.rotating_logfile_parameters.log_prefix = WorkerLogPrefix.Get();
+		Logsink.rotating_logfile_parameters.max_log_files = 1;
+		Logsink.rotating_logfile_parameters.max_log_file_size_bytes = 10485760; // 10 MB
```

Comment:
Seems arbitrary. Can't we we just make this infinite, users never want to lose logs.

Reply:
Yeah, true. The thing is that the Worker SDK currently doesn't support infinite file size. That was an oversight on my part, as I we always had rotating log files with the old protocol logging and I was under the assumption that this was what everyone wanted. I can raise a PR with Worker to add infinite log file sizes (for a single file). Until then, I could add a runtime setting?

Reply:
Woah hardcore I love it, make PR's into their code when issues arise <3
Let's perhaps make this a user option until infinite is available, at least then if someone needs more than 10MB of last data they can set it?

Reply:
Since I wrote most of that stuff on Worker side it might actually be faster if I just raise a PR 😄 

I'll add a user option on Unreal side then

Reply:
Opened PR with Worker: https://github.com/improbable/everything/pull/5268

Also, added more command line parameters to control the log size in the meantime.